### PR TITLE
Use cookies to remember language of choice

### PR DESCRIPTION
**Pull request by dpgraham**
* Use cookies to remember language of choice
    * Added getter and setter for cookies
    * Sets the active language on document ready (Java by default)
    * Bound to selection event so that it updates the default language
* Also includes a tab sorting fix
    * Languages that aren't on the priority list were being shown first
    * Added objectivec to list

### CODE REVIEW

**Review by jlipps — APPROVED** [review]
looks great, and not too complex either! awesome

File: test/tabs-specs.js [hunk 1]
```diff
@@ -116,6 +116,10 @@ describe('Repo.js', function () {
 
       const html = fencedCodeTabify(`<div>
         <pre>
+          <code class='prolog'>// Prolog
```
Comment (mykola-mokhnach) [inline test/tabs-specs.js:119]: hehe oldschool

File: cinder/js/extra.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable */
```
Comment (mykola-mokhnach) [inline cinder/js/extra.js:1]: should we also declare `use strict`?

File: test/tabs-specs.js (hunk 1 above)
Comment (dpgraham) [reply test/tabs-specs.js:119]: Haha. It's making a comeback!

**Pull request merged**